Aztec/Classes/Extensions/NSAttributedString+Lists.swift

```diff
@@ -63,13 +63,82 @@ extension NSAttributedString
     func rangeOfTextList(atIndex index: Int) -> NSRange? {
         var effectiveRange = NSRange()
         let targetRange = rangeOfEntireString
-        guard let _ = attribute(TextList.attributeName, at: index, longestEffectiveRange: &effectiveRange, in: targetRange) as? TextList else {
+        guard
+            let paragraphStyle = attribute(NSParagraphStyleAttributeName, at: index, longestEffectiveRange: &effectiveRange, in: targetRange) as? ParagraphStyle,
+            let _ = paragraphStyle.textList
```

Comment:
Can we remove this line?. If we're relying on an internal initialization this triggers, can we add a comment?